src/Uno.UI.RuntimeTests/Tests/NativeCtorGeneratorTests/GameBoardCanvas.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if MONOANDROID
```

Comment:
You can use 
```suggestion
﻿#if __ANDROID__
```
or suffix the file with `.android.cs`